content/applications/general/in_app_purchase.rst

```diff
@@ -1,59 +1,179 @@
 =====================
-In-App Purchase (IAP)
+In-app purchase (IAP)
 =====================
 
-In-App Purchases (IAP) gives access to additional services through Odoo. For instance, it allows you
-to send SMS Text Messages or to send Invoices by post directly from your database.
+In-app purchases (IAP) are optional services that enhance Odoo databases. Each service provides its
+own specific features and functionality.
 
```

Comment:
You should move the iap.png image up to here -- between the intro and the example admonition

Reply:
and remove it from its current position (obviously haha)